testsuite/features/step_definitions/api_common.rb

```diff
@@ -312,6 +312,13 @@
     child_channels.reject! { |channel| channel.include? 'suse-manager-retail-branch-server-5.0-updates-x86_64' }
   end
 
+  # filter out wrong child channels for SL Micro 6.1 as normal Minion
+  if client.include? 'slmicro61'
```

Comment:
I wonder if we don't reproduce the same issue with SLES15SP7, when we are on a BV where the HostOS of the server is a SLES15SP7.